HealthcareSystem/PatientWebAppUnitTests/CreateObjectsForTests/ICreateObject.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace PatientWebAppTests.CreateObjectsForTests
+{
+    public interface ICreateObject
+    {
+        object createValidObject();
+        object createInvalidObject();
```

Comment:
Why object?

Reply:
What return value should I put if I want this factory to create different types of objects? If the solution is to separate factories by objects, then instead of 2 methods for creating valid and invalid objects, I will have 4 classes for each type of object. I think it is overkill.

Reply:
The method should do one thing, not two, only because of that I suggest separation. But if you think that is not good, okay, this is still your code.